### PR TITLE
Mahdiyeh/fix: fix destroyed dialog error on workplace

### PR DESCRIPTION
**Pull request by mahdiyeh-fs**
in the current version of webtrader we add destroyed dialog to the workplace table, so whenever we click on the table rows we get error which refers that this dialog is not initialized yet. In this PR the function `addDialog` is called on dialogOpen and `removeDialog` is called whenever the dialog get closed.

### CODE REVIEW

File: src/workspace/workspace.es6 [hunk 1]
```diff
@@ -267,9 +267,10 @@ export const init = (parent) => {
    parent.append(root);
    rv.bind(root[0], state);
 }
-export const addDialog = (name, clickCb, removeCb) => {
+export const addDialog = (name, id, clickCb, removeCb) => {
```
Comment (matin-binary) [inline src/workspace/workspace.es6:270]: Why did you add `id` as the second argument to this function?
Are we passing the value once we invoke the function?

Comment (mahdiyeh-fs) [reply src/workspace/workspace.es6:270]: we need this id for removeDialag function. yes we are.

Comment (matin-binary) [reply src/workspace/workspace.es6:270]: Where do we invoke the function from?

Comment (mahdiyeh-fs) [reply src/workspace/workspace.es6:270]: in windows.es6 file line 373

**Review by matin-binary — APPROVED** [review]
